acceptance/setup/common/050_Setup_Broker.rb

```diff
@@ -55,7 +59,7 @@
   # some time and will eat into the polling period we allow for the broker to start
   for i in 0..NUM_BROKERS-1
     lein_command = "cd #{GIT_CLONE_FOLDER}/pcp-broker#{i}; export LEIN_ROOT=ok; lein with-profile #{LEIN_PROFILE} deps"
-    lein_command = append_jvm_limits(lein_command) if master[:gke_container]
+    lein_command = append_jvm_opts('-Xms2g -Xmx2g', lein_command) if master[:gke_container]
```

Comment:
ah, i see this is where that method is used. I think its simple enough just to conditionally append it here without using a general method https://github.com/puppetlabs/pxp-agent/pull/835/files#r701994227